src/processor/tree/tree_wrapper.rs

```diff
@@ -8,12 +8,21 @@ use state_reconstruct_fetcher::{
     database::InnerDB,
     types::{v2::PackingType, CommitBlock},
 };
+use thiserror::Error;
 use tokio::sync::Mutex;
 use zksync_merkle_tree::{Database, MerkleTree, RocksDBWrapper, TreeEntry};
 
 use super::RootHash;
 
+#[derive(Error, Debug)]
+pub enum TreeError {
+    #[error("block mismatch")]
+    BlockMismatch,
+}
+
 pub struct TreeWrapper {
+    index2key: HashMap<u64, U256>,
+    key2value: HashMap<U256, H256>,
```

Comment:
```suggestion
    index_key_map: HashMap<u64, U256>,
    key_val_map: HashMap<U256, H256>,
```
Nit: This is matter of personal preference, but I prefer this naming convention

Reply:
well, clippy doesn't complain...

Reply:
but that's a different object - a bigger one, for a start...

Reply:
Well, it's not really the object that's the focus, more the naming convention. Though, in the end I don't mind all too much, but I would prefer to have things be somewhat consistent if that's all right with you

Reply:
well, you know what they say about "foolish consistency", but OK...